auto_dev/contracts/block_explorer.py

```diff
@@ -32,7 +32,9 @@ def get_abi(self, address: str) -> Optional[Dict]:
             ValueError: If the response is invalid or missing ABI data
         """
         try:
-            url = f"{self.url}/{address}?network={self.network}"
+            url = f"{self.url}/{address}"
+            if self.network != "ethereum":
+                url += f"?network={self.network}"
             response = requests.get(url, timeout=DEFAULT_TIMEOUT)
```

Comment:
use the standard functions of the requests network, i.e. params = dictionary. 
Otherwise open to user input attacks